### ISSUE TITLE
Help in creating USB Boot

### ISSUE THREAD

**Issue by Spider84** [DavidXanatos/DiskCryptor#1]
Hello!
Create work! 
I installed EFI Bootloader to USB flash and OS booted fine from USB flash BUT i need to pres V (Boot Crypt loader from rescue disk). How to chenge it to autoboot?

**Comment by DavidXanatos** [comment]
So you don't want to install the Bootloader on the HDD you want it to run from the USB but not show recovery options.

Assuming you have a 64 bit system, to get this behavior:
Replace \EFI\Boot\BOOTx64.efi with \EFI\DCS\DcsBoot.efi

**Comment by Spider84** [comment]
Thx for quick answer!
As I undestand I need to copy files \EFI\Boot\Microsoft partition from HDD to USB too.

**Comment by DavidXanatos** [comment]
No, the UEFI bootloader looks for the windows boot manager on the drives it finds an encrypted partition on.

**Comment by Spider84** [comment]
Yes. I already found this algorithm in source code.
Now I encoded EFI and system partitions and all work fine, but was needed to rebuild Microsoft\Boot\BCD by bcdbuild.exe
I plan to write step by step manual - at first for myself, second for all others :)

**Comment by DavidXanatos** [comment]
Cool :-)

**Comment by YuriRamos2** [comment]
How about reaching out to https://www.hirensbootcd.org/ so they can include it on their builds? I've already used it to fix bootloader after Windows upgrade, but since you can't start without rebooting I cheated by removing an usb drive and mounting it again, after that Diskcryptor stops asks for a reboot. But using it like this limit uses (I can't decrypt an harddisk).
